src/components/NavBar/NavigationBar.tsx

```diff
@@ -35,24 +46,44 @@ function NavigationBar(): JSX.Element {
     setIsNavVisible(!isNavVisible);
   }
 
-  const {t} = useTranslation('menu');
+  /** Toggle the language of the website between English and Spanish. Close menu bar after toggling if on mobile. */
+  function toggleLanguage(): void {
+    i18n.changeLanguage(i18n.language === 'en' ? 'es' : 'en');
+    if (isMobile) {
```

Comment:
nit: you can put this on one line: `if (isMobile) toggleNav();`

Reply:
Do we do this anywhere else in our code? I feel like at least in my personal style guide, it's always better to do multi-line if statements because then if one later makes it longer, missing curly braces don't cause incorrect behavior.

Reply:
Keeping per small conversation right after Carson left on Wednesday (sorry Carson)